### DISCUSSION TITLE
What is the second column in the fs.ls output?

### DISCUSSION THREAD

**Discussion by CKHH** [seaweedfs/seaweedfs#5428] [Q&A] [answered]
Hello,

Q1: In weed shell, when `fs.ls -la`, what is the meaning of value `1` in second column (right after `-rw-r--r--`) ?
```
-rw-r--r--   1    10585 /buckets/fir/uat/24-03-26/test1.pdf
-rw-r--r--   1    11077 /buckets/fir/uat/24-03-26/test2.pdf
-rw-r--r--   1     4686 /buckets/fir/uat/24-03-26/test3.txt
-rw-r--r--   1     7991 /buckets/fir/uat/24-03-26/test4.txt
```

Q2: Why the value always change to `0` when `fs.ls -la` again later (after some time) ?

Thanks.

**Comment by eliphatfs** [comment, marked as answer]
Q1: The number of chunks in the file.
```go
fmt.Fprintf(writer, "%s %3d %s %s %6d %s/%s\n",
				fileMode, len(entry.GetChunks()),
				userName, groupName,
				filer.FileSize(entry), dir, entry.Name)
```

I have no idea why it would change to 0.

**Reply by CKHH** [reply]
Thanks @eliphatfs . 
I found out that it would change to 0 after the `remote.uncache` execution in master maintenance.
